ember-bootstrap/addon/components/bs-navbar.js

```diff
@@ -13,10 +13,8 @@ import { isBlank } from '@ember/utils';
 
   ```hbs
   <BsNavbar as |navbar|>
-    <div class="navbar-header">
-      <navbar.toggle />
-      <a class="navbar-brand" href="#">Brand</a>
-    </div>
+    <a class="navbar-brand" href="#">Brand</a>
+    <navbar.toggle />
```

Comment:
`.navbar-brand` link is before the toggle button in [Bootstrap docs](https://getbootstrap.com/docs/5.3/components/navbar/). I switched the order here accordingly.